src/impl.rs

```diff
@@ -591,6 +593,16 @@ impl NpmBuild {
     pub fn to_resource_dir(self) -> ResourceDir {
         self.into()
     }
+
+    #[cfg(not(windows))]
+    fn command(&self) -> Command {
+        Command::new(&self.executable)
+    }
+
+    #[cfg(windows)]
+    fn command(&self) -> Command {
+        Command::new("cmd").arg("/c").arg(&self.executable)
```

Comment:
This doesn't compile on windows:

```
   --> src/impl.rs:604:9
    |
603 |     fn command(&self) -> Command {
    |                          ------- expected `std::process::Command` because of return type
604 |         Command::new("cmd").arg("/c").arg(&self.executable)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `std::process::Command`, found `&mut std::process::Command`
```

However, the following compiles just fine:

```rust
#[cfg(windows)]
fn command(&self) -> Command {
    let mut cmd = Command::new("cmd");
    cmd.arg("/c").arg(&self.executable);
    cmd
}
```

Reply:
@idursun thanks! I should not develop anything late evening, just taking your code now.

Reply:
checked on my windows machine